resources/dicts/thoughts/alive/general.json

```diff
@@ -51,87 +49,1424 @@
+            "Tried to fight a kittypet",
+            "Tried to fight a rogue",
+            "Wonders if training in the Dark Forest is really that bad",
+            "Daydreams about killing a certain cat",
+            "Wonders if {PRONOUN/m_c/subject} could kill a fox all by {PRONOUN/m_c/self}",
+            "Started a fight with a fellow Clanmate",
+            "Wonders how {PRONOUN/m_c/subject} would run the Clan better",
+            "Licks fresh-kill blood off {PRONOUN/m_c/poss} lips a little too intently",
+            "Has been glaring at a certain cat in camp with an odd look in {PRONOUN/m_c/poss} eye",
+            "Wonders how easy it'd be to attack the leader from where they lay",
+            "Daydreams about slaughtering every cat {PRONOUN/m_c/subject} {VERB/m_c/despise/despises}"
+        ],
+        "main_trait_constraint": [
+            "bloodthirsty"
+        ]
+    },
+    {
+        "id": "bold_gen",
+        "thoughts": [
+            "Is getting some looks after speaking out of turn at the last Clan meeting",
+            "Spoke out of turn at the last Clan meeting",
+            "Bravely offers to help the medicine cat pick herbs from a dangerous spot",
+            "Runs out of {PRONOUN/m_c/poss} den into 'battle', only to find that it was just a storm",
+            "Prances around with {PRONOUN/m_c/poss} tail up high, ready for anything",
+            "Promises the kits {PRONOUN/m_c/subject}'ll protect them when the time comes",
+            "Is not afraid of anything",
+            "Is always first to volunteer when a dangerous mission is announced"
+        ],
+        "main_trait_constraint": [
+            "bold"
+        ]
+    },
+    {
+        "id": "calm_gen",
+        "thoughts": [
+            "Is taking a moment to appreciate some peace and quiet",
+            "Is patiently listening to someone talk about things they are unquestionably wrong about and dying inside",
+            "Looks on as the camp buzzes around {PRONOUN/m_c/object}",
+            "Is basking in the gentle sunlight around camp",
+            "Is meditating upon a sun-warmed rock",
+            "Is sitting quietly on the far side of camp",
+            "Is humming a soothing tune to {PRONOUN/m_c/self} as {PRONOUN/m_c/subject} {VERB/m_c/work/works}",
+            "Is out on a peaceful stroll",
+            "Is remembering simpler days with fondness",
+            "Is offering gentle advice to a frustrated Clanmate",
+            "Is lining {PRONOUN/m_c/poss} nest with lavender",
+            "Smells sweet, like lavender",
+            "Never fails to offer comfort to {PRONOUN/m_c/poss} Clanmates",
+            "Is taking {PRONOUN/m_c/poss} sweet time eating {PRONOUN/m_c/poss} meal",
+            "Is purring quietly",
+            "Has been reaching meditative states while sunbathing",
+            "Calmly takes the lead while some Clanmates argue over this and that",
+            "Is unable to be scared, no matter how much the kits try",
```

Comment:
should only happen if there _are_ kits?

Reply:
definitely right about that! fixing soon